### DISCUSSION TITLE
Default logging documentation

### DISCUSSION THREAD

**Discussion by damaon** [celery/celery#9378] [General]
I've spent considerable amount of time to have some sane logic setup for Celery and I've tried to find in documentation how to setup something like this:

`PID[562] ID[f4a47aa9-ba5e-48ae-bcec-ce7347c4a113] JOB[refresh_cache] FROM[/app/workers/worker.py:53] [INFO] cache.py:59`

Basically: process.id, task_id, name of job, who called delay/apply_async, type of msg, log filename

Why something similar isn't the default? I wasn't able to easily find how to achieve these things and it seems to be like basics for debugging stuff. Docs for reference - https://docs.celeryq.dev/en/stable/userguide/tasks.html#logging

I've used before_task_publish, custom logging.Filter and formatter to achieve it in case someone hits similar issue.

**Comment by iloveitaly** [comment]
Is there a place where I can extract the PID or worker ID from a particular worker pool?

I'd like to figure out where the `8` comes from in the following default log:

```
WARNING/ForkPoolWorker-8
```
